web/impact/impact/graphql/types/startup_type.py

```diff
@@ -27,3 +28,8 @@ def resolve_name(self, info, **kwargs):
     def resolve_high_resolution_logo(self, info, **kwargs):
         if self.high_resolution_logo:
             return self.high_resolution_logo.url
+
+    def resolve_program_startup_status(self, info, **kwargs):
+        return self.program_startup_statuses().filter(
+            startupstatus__startup=self).values_list(
```

Comment:
you don't need to add a filter:`self.program_startup_statuses().filter(startupstatus__startup=self)` is the same as `self.program_startup_statuses()`